app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -253,7 +270,136 @@ function toGqlChatCompletionRole(
   }
 }
 
-function PlaygroundOutputText(props: PlaygroundInstanceProps) {
+function NonStreamingPlaygroundOutputText(props: PlaygroundInstanceProps) {
+  const instances = usePlaygroundContext((state) => state.instances);
```

Comment:
this is just copied from streaming, can probably make a wrapper and just change the subscription or mutation, just wanted to get something out